libs/ngrx-toolkit/src/lib/devtools/with-tracked-reducer.ts

```diff
@@ -0,0 +1,64 @@
+import { inject, untracked } from '@angular/core';
+import {
+  EmptyFeatureResult,
+  getState,
+  PartialStateUpdater,
+  SignalStoreFeature,
+  signalStoreFeature,
+  type,
+} from '@ngrx/signals';
+import { ReducerEvents, withEventHandlers } from '@ngrx/signals/events';
+import { tap } from 'rxjs/operators';
+import { updateState } from './update-state';
+
+export function withTrackedReducer<State extends object>(
+  ...caseReducers: CaseReducerResult<State, any>[]
```

Comment:
For some reason, I had to use `any` here. The `EventCreator` - as it is in the original version - didn't work. I guess that this is due to different settings in the `tsconfig.json`. 

Unless, we won't get meta reducers, I wouldn't investigate this any further. The only thing I would change is to replace `any` with `unknown`.